activerecord/lib/active_record/encryption/config.rb

```diff
@@ -39,6 +40,7 @@ def set_defaults
           self.excluded_from_filter_parameters = []
           self.previous_schemes = []
           self.forced_encoding_for_deterministic_encryption = Encoding::UTF_8
+          self.add_to_filter_attributes = false
```

Comment:
I made the default `false` for backwards compatibility. 

- is this correct?
- should we make it true in `new_framework_defaults`?